packages/@n8n/nodes-langchain/nodes/agents/Agent/Agent.node.ts

```diff
@@ -113,6 +113,7 @@ function getInputs(
 						'@n8n/n8n-nodes-langchain.lmChatAnthropic',
 						'@n8n/n8n-nodes-langchain.lmChatAzureOpenAi',
 						'@n8n/n8n-nodes-langchain.lmChatMistralCloud',
+						'@n8n/n8n-nodes-langchain.lmChatOllama',
```

Comment:
tangential thought: instead of maintaining arrays of nodetypes like these, could we instead add a new property on AI nodes called `capabilities` that could define what kind of special inputs/outputs that particular nodes can attach to. That could help decouple nodes better in the long term.
That could also then be used by community AI nodes. So we don't have to create and maintain a chat node for every service out there.

Reply:
I like that! Added it to our AI board as tech-debt. We plan to plan & tackle some of these next month